test/e2e/storage/subpath.go

```diff
@@ -480,7 +468,7 @@ func testPodContainerRestart(f *framework.Framework, pod *v1.Pod, fileInVolume,
 	Expect(err).ToNot(HaveOccurred())
 
 	By("Failing liveness probe")
-	out, err := podContainerExec(pod, 1, fmt.Sprintf("rm %v", fileInVolume))
+	out, err := podContainerExec(pod, 1, fmt.Sprintf("rm %v", probeFilePath))
 	framework.Logf("Pod exec output: %v", out)
 	Expect(err).ToNot(HaveOccurred())
```

Comment:
Sometimes when debugging e2e tests, I found it useful to print why a particular expect failed. For example:

```
Expect(err).ToNot(HaveOccured(), "While waiting for pod exec to report error on %s case", "blah")
```

Usually when we are processing e2e logs, it gives us clear indication of which scenario is failing.